saml2/saml2-service-provider/src/main/java/org/springframework/security/saml2/provider/service/registration/RelyingPartyRegistration.java

```diff
@@ -542,6 +548,15 @@ public Saml2MessageBinding getSingleSignOnServiceBinding() {
 			return this.singleSignOnServiceBinding;
 		}
 
+		/**
+		 * Return the algorithm URI used to sign XML document
```

Comment:
It would be nice if the JavaDoc here explained how this value relates to the asserting party's `<IDPSSODescriptor>`.